packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/ParametricSVGImageView.tsx

```diff
@@ -91,6 +91,31 @@ export class ParametricSVGImageView extends RectangularNodeView {
     if (labelIndex > -1) {
       nodeLabel = node.children.at(labelIndex) as SLabel;
     }
+    const debugFontSize = 6;
+
+    const styleDebug = {
+      fontSize: debugFontSize + 'px',
+      fill: 'green',
+    };
+
+    const debugInfos = (node: Readonly<Node>, rectangleSize: { width: number; height: number }) => {
+      const diagram: Diagram = node.root as Diagram;
+      if (diagram.debug) {
+        return (
+          <g>
+            <text x={0} y={0 + debugFontSize} style={styleDebug}>
+              x:{node.bounds.x.toFixed(2)} y:{node.bounds.y.toFixed(2)}
+            </text>
+            <text x={rectangleSize.width / 2} y={rectangleSize.height - debugFontSize} style={styleDebug}>
+              Width: {rectangleSize.width.toFixed(2)}
+            </text>
+            <text x={0} y={rectangleSize.height - debugFontSize} style={styleDebug}>
+              Height: {rectangleSize.height.toFixed(2)}
+            </text>
+          </g>
+        );
+      } else return null;
+    };
```

Comment:
You can share the same code instead of re-creating it from scratch in all components